### ISSUE TITLE
Typings don't work when cborg is imported in a ESM project

### ISSUE THREAD

**Issue by Aenigma** [rvagg/cborg#77]
I'm using an ESM module project to import cborg and I got this error trying to use it from VS Code.

```
Could not find a declaration file for module 'cborg'. '~/projects/my-project/node_modules/cborg/esm/cborg.js' implicitly has an 'any' type.
  There are types at '~/projects/my-project/node_modules/cborg/types/cborg.d.ts', but this result could not be resolved when respecting package.json "exports". The 'cborg' library may need to update its package.json or typings.ts(7016)
```

I could make the error go away by changing the `exports` in `package.json` and I'm happy to PR that change, but I'm not sure what exactly best practices are in this realm.

**Comment by BANOnotIT** [comment]
I can't really use this package in my vite project. Typescript just rejects to import typings not from exports.

**Comment by rvagg** [comment]
See https://github.com/rvagg/cborg/pull/91, you can help test by installing `cborg@3.0.0-alpha.1` and seeing if it works for you.

**Comment by rvagg** [comment]
should be fixed by 3.0.0
